dll/win32/shell32/CDefView.cpp

```diff
@@ -1663,7 +1658,7 @@ LRESULT CDefView::OnPrintClient(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &b
     }
     else
     {
-        FillRect(hDC, &rc, GetSysColorBrush(COLOR_WINDOW));
+        SHFillRectClr(hDC, &rc, GetViewColor(m_viewinfo_data.clrTextBack, COLOR_WINDOW));
```

Comment:
Why not continuing using FillRect here?

Reply:
HBRUSH vs COLORREF. The old code was never updated to respect `m_viewinfo_data.clrTextBack`.